src/CashMachine.java

```diff
@@ -0,0 +1,55 @@
+import java.nio.channels.ScatteringByteChannel;
+import java.util.Scanner;
+
+public class CashMachine {
+    public static void main(String[] args) throws NoFundsException, NotAllowedTransactionException {
+
+        Person person1 = new Person("jan", "kowalski", "1234");
+        Person person2 = new Person("adam", "nowak", "5678");
+
+        BankAccount[] accounts = new BankAccount[2];
+
+        try {
+            accounts[0] = new BankAccount(person1, 1000);
+            accounts[1] = new BankAccount(person2, 500);
+            System.out.println(accounts[0].printInfo());
+            System.out.println(accounts[1].printInfo());
+        } catch (NullPointerException e) {
```

Comment:
lapanie wyjatku `NullPointerException` to zla praktyka. mozemysz przez przyapadek zlapac uzice innej referencji ktora przez przypadek jest nullem

Reply:
OK. Usunięte.